ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -61,14 +61,13 @@ enum CocoaPodUtils {
     let subspecs: Set<String>
 
     /// The contents of the module map for all frameworks associated with the pod.
-    var moduleMapContents: String
+    var moduleMapContents: ModuleMapBuilder.ModuleMapContents?
```

Comment:
Why changed to Optional? (sorry if I'm missing it)

Reply:
It was initialized to empty string before - that's no longer possible since the type is no longer String.  It's an improvement to have an uninitialized struct versus a wrong empty string.

Reply:
We probably need a bigger refactor since the initialization of moduleMapContents is still ugly - but that's outside the scope of current PR